src/types.js

```diff
@@ -24,8 +24,8 @@ export type Dimensions = {|
 |};
 
 export type InputSelectionType = {|
-  start: number,
-  end: number,
+  +start: number,
```

Comment:
Duh, this is the most subtle thing a diff can show `-  start` vs `+  +start`.

Reply:
I'm not sure if that's a request to do something different, or just a lament about Flow's syntax. :slightly_smiling_face: 

I do think this is a good reason to keep this change separate from large other changes, or other changes to this chunk of code! As is, though, as the reader you know *something* changed in this block because there's a diff shown here... so it should get your attention, and then hopefully it isn't too hard to spot what.

One other tool that helps: `diff-highlight`. See my .gitconfig for how to enable it:
https://gist.github.com/gnprice/5f71b63464a76bd7e0480f207b1bbff3
The tool itself is distributed with Git; the Debian package puts it in `/usr/share/doc/git/contrib/diff-highlight`, and in general `locate diff-highlight` should find it on your system. In cases like this, it causes the one small piece of each line that actually changed to get highlighted in `git diff`, `git log -p`, etc.

![image](https://user-images.githubusercontent.com/28173/56317647-17e99380-6112-11e9-865a-8d3e35b19977.png)

The next commit is actually a yet stronger example of where `diff-highlight` is super helpful -- every one of the diff hunks has the word `Type` in red, which helps me see at a glance that the exact same boring thing is happening in each one of them.

![image](https://user-images.githubusercontent.com/28173/56317960-e2917580-6112-11e9-9c4f-54da4d460c8a.png)